setup.cfg

```diff
@@ -61,3 +61,25 @@ ignore =
 
 [devpi:upload]
 formats = sdist.tgz,bdist_wheel
+
+[mypy]
+# check_untyped_defs = True
+# disallow_any_decorated = True
+# disallow_any_explicit = True
+# disallow_any_expr = True
+# disallow_any_generics = True
+# disallow_any_unimported = True
+# disallow_incomplete_defs = True
+# disallow_subclassing_any = True
+# disallow_untyped_calls = True
+# disallow_untyped_decorators = True
+# disallow_untyped_defs = True
+ignore_missing_imports = True
```

Comment:
when run through `pre-commit` this will be the default so you can remove it

Reply:
I'd prefer to keep it so that editor/manual mypy can pick it up.